src/Microsoft.OData.Edm/Csdl/CsdlWriter.cs

```diff
@@ -56,6 +60,17 @@ public static bool TryWriteCsdl(IEdmModel model, Utf8JsonWriter writer, out IEnu
             return TryWriteCsdl(model, writer, CsdlJsonWriterSettings.Default, out errors);
         }
 
+        /// <summary>
+        /// Outputs a CSDL JSON artifact to the provided <see cref="Utf8JsonWriter"/> Asynchronously.
+        /// </summary>
+        /// <param name="model"></param>
+        /// <param name="writer"></param>

```

Comment:
Complete the doc strings.

Reply:
Do this in all other places.

Reply:
@ElizabethOkerio I have added params description across other places as well.